day_6/exercises/burrito.rb

```diff
@@ -5,12 +5,25 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_reader :protein, :base, :toppings)
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
```

Comment:
Hey Daniel! This is close to what we are looking for on this exercise, but unfortunately the += and -= syntax does not work for arrays. However! There are built in array methods that one can utilize in ruby that are not that far off what you have here. Can you find a method that will accomplish this in the docs? https://ruby-doc.org/core-2.4.1/Array.html